### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by richardzeng1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,115 @@
...
+
+```
+npm start
+```
+
+Note: This starts a PHP server on port 3000.
+
+## Database Setup
+
+Login to database via Mariadb (I used MYSQL client via windows)
```
Comment (F-Ting) [inline README.md]: Remove the parenthesis here

File: README.md [hunk 2]
```diff
@@ -1,2 +1,115 @@
...
+DROP DATABASE mydb;
+```
+
+#### Now that the Database is created, you could use the following command to see the list of databases that are available
+
+```
+MariaDB[(none)]> SHOW DATABASES;
+```
+
+Select the database that's being used
```
[the remark this answers is not in the document]
Comment (david-yz-liu) [reply README.md]: Also, either refer to Step 1 above, or use the `mydb` name in the command below.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,115 @@
...
+
+```
+MariaDB[(none)]> USE database_name;
+```
+
+Once the database is selected, import the db/schema.sql file
```
Comment (david-yz-liu) [inline README.md]: The path `db/schema.sql` is inconsistent with #9. Probably that file should be moved (I like `db/schema.sql` better).

File: README.md [hunk 4]
```diff
@@ -1,2 +1,115 @@
...
+
+```
+MariaDB[database_name]> source "/absolute/path/to/.../db/schema.sql"
```
Comment (david-yz-liu) [inline README.md]: Also, I just tried using a relative path and it seemed to work (on Windows). Does this not work on OSX/Linux?

File: README.md (hunk 3 above)
Comment (richardzeng1) [reply README.md]: That is changed in the updated pull request Darren submitted.

File: README.md (hunk 4 above)
Comment (richardzeng1) [reply README.md]: It works on OSX

Comment (richardzeng1) [reply README.md]: Relative path works but the absolute path might reduce potential errors? Is this a problem?

**Pull request merged**